rules/do-you-know-how-to-claim-expense-reimbursements-2/rule.md

```diff
@@ -3,27 +3,23 @@ type: rule
 archivedreason: 
 title: Do you know how to claim expense reimbursements with Xero Expense app
 guid: 2e71cd53-d456-45bc-8a89-5ad9948cd85b
-uri: do-you-know-how-to-claim-expense-reimbursements-2
```

Comment:
Add this to the redirects so if people try to visit the old url they are taken to the new one instead of receiving an error

Reply:
Done - I added this to the list of redirects. I am not sure if the uri needs changing, however, I have left it as is.